chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
```

Comment:
I think `value1`, `value2` and `message` would be more consistent with the rest of the definition file.